http/src/request/channel/reaction/mod.rs

```diff
@@ -11,11 +11,21 @@ pub use self::{
     get_reactions::GetReactions,
 };
 use std::fmt::Write;
-use twilight_model::id::EmojiId;
+use twilight_model::{channel::ReactionType, id::EmojiId};
 
+#[derive(Eq, PartialEq)]
```

Comment:
I'm a little on the fence about the equality check here. One Custom instance that has an ID and a name but another Custom instance that has an ID but no name would not be equal. Should we check for identifier equality or complete content equality?

Reply:
I guess the rest of the crates go with content equality, so I guess this is fine.